src/components/Calendar/index.js

```diff
@@ -69,13 +183,94 @@ class CalendarComponent extends Component {
         const newMonth = new Date(this.state.currentMonth);
         newMonth.setFullYear(year);
         this.setState({
+            focusedDate: getFirstDayMonth(newMonth),
             currentMonth: newMonth,
         });
     }
 
+    handleKeyDown(event) {
+        const { keyCode, altKey } = event;
+        const keyHandler = altKey ? this.keyHandlerMapAlt : this.keyHandlerMap;
+        if (keyHandler[keyCode]) {
+            event.preventDefault();
+            event.stopPropagation();
+            keyHandler[keyCode]();
+        }
+    }
+
+    handleKeyUpPressed() {
+        this.moveFocusedDay(-7);
+    }
+
+    handleKeyDownPressed() {
+        this.moveFocusedDay(7);
+    }
+
+    handleKeyLeftPressed() {
+        this.moveFocusedDay(-1);
+    }
+
+    handleKeyRightPressed() {
+        this.moveFocusedDay(1);
+    }
+
+    handleKeyHomePressed() {
+        const { focusedDate } = this.state;
+        this.moveFocusedDay(-focusedDate.getDay());
+    }
+
+    handleKeyEndPressed() {
+        const { focusedDate } = this.state;
+        this.moveFocusedDay(6 - focusedDate.getDay());
+    }
+
+    handleKeyPageUpPressed() {
+        this.moveFocusedMonth(-1);
```

Comment:
there is a problem when try to change month or year when minDate and maxDate are not passed